cmd/snapshot-controller/main.go

```diff
@@ -100,11 +103,18 @@ func ensureCustomResourceDefinitionsExist(client *clientset.Clientset) error {
 		return true, nil
 	}
 
-	// with a Factor of 1.5 we wait up to 7.5 seconds (the 10th attempt)
+	maxMs := retryCRDIntervalMax.Milliseconds()
+	if maxMs < 100 {
+		maxMs = 100
+	}
+	steps := int(math.Ceil(math.Log(float64(maxMs)/100) / math.Log(1.5)))
```

Comment:
This 1.5 here is the same as Factor 1.5 on line 116?  Can you use the same parameter if they are the same?

Reply:
Yup, explained in the comment.